dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/security/DefaultSecurityService.java

```diff
@@ -673,4 +675,16 @@ public boolean canDataRead( IdentifiableObject identifiableObject )
         return !aclService.isSupported( identifiableObject.getClass() )
             || aclService.canDataRead( currentUserService.getCurrentUser(), identifiableObject );
     }
+
+    private String encodeUserName( String userName )
```

Comment:
This seems to be a very generic method for utf8 encoding strings. I suggest let's rename this method to `utf8Encode` and move it to class `CodecUtils`.

Reply:
agree, fixed.